e2e_testing/torchscript/reduction.py

```diff
@@ -122,3 +122,63 @@ def forward(self, a):
 @register_test_case(module_factory=lambda: ReduceMeanDtypeModule())
 def ReduceMeanDtypeModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 4, 5).to(torch.float64))
+
+# ==============================================================================
+
+class ReduceMaxAlongDim(torch.nn.Module):
+
+  def __init__(self):
+    super().__init__()
+
+  @export
+  @annotate_args([
+      None,
+      ([-1, -1, -1], torch.float64, True),
+  ])
+  def forward(self, a):
+    return torch.max(a, 1)[0]
```

Comment:
nit: It's better to use `torch.ops.aten.max.dim` directly so that it's more obvious that's the variant being tested.